CLUE_BBQ/clue_bbq.py

```diff
@@ -122,27 +123,28 @@
 
                 temps = ibbq_service.temperatures
                 batt = ibbq_service.battery_level
-                if temps is not None:
+                if temps != None:
                     probe_count = len(temps)  # check how many probes there are
                     for i in range(probe_count):
-                        if (temps[i] is not 0 and temps[i] < 1000):  # unplugged probes
+                        if temps[i] is not 0 and temps[i] < 1000:  # unplugged probes
```

Comment:
Sorry if I wasn't clear. It should be `temps is not None` on line 126. and `temps[i] != 0` on line 129.

Reply:
ah, OK, got it.